README.md

```diff
@@ -0,0 +1,28 @@
+# di-account-management-backend
+
+The backend and data store that serves the account management application.
+
+This is a serverless application for AWS. It's built and deployed using the [SAM CLI](https://aws.amazon.com/serverless/sam/).
```

Comment:
Should we specify the Node version here ?

Reply:
That's a good point - I'm not sure where tells you that we're using node 16 currently... I think we do need that somewhere, But perhaps not here... if we need to keep it up to date I reckon we'd forget to change the README.

Perhaps we should look at adding `.nvmrc`'s to each lambda repo ([link here](https://github.com/nvm-sh/nvm#nvmrc)) or is there something we can specify in package.json?

Then perhaps we could have a per-requsites heading here about things you'd need to install to get that running.
I'd suggest we do anything like that in a followup PR though.

Reply:
Good point. You'd have to look at the Cloudformation template to see what version of Node we're deploying with. Not ideal...

We recommend using `nvm` in the new onboarding guide, so I think it makes sense to use the tools for that (`.nvmrc` files). That would mean we can just run 

```bash
nvm install
nvm use
```

in a lambda folder and get the right version. Much nicer than putting a directive in `package.json` which will tell you you've got the wrong version but not give you a way to install the correct one.

I'll do a follow up PR to add those in and put some instructions in the README